sdk_lib/make_chroot.sh

```diff
@@ -47,6 +47,11 @@ DEFINE_string stage3_path "" \
   "Use the stage3 located on this path."
 DEFINE_string cache_dir "" "Directory to store caches within."
 
+DEFINE_string extra_portdir_overlay_path "" \
+  "Extra PORTDIR_OVERLAY direcory path outside of the chroot."
+DEFINE_string extra_portdir_overlay_name "" \
```

Comment:
I did try to make sure the options help text was explicit enough, but your right, testing the assumption would be better, I was just worried about adding too much logic to the bash scripts.